tools/deb/istio-iptables.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
```

Comment:
Sure. But atleast provide the diff or comment container lines and add vm lines so that others can understand what is different here vs vm. I would like to review these rules but it's going to take a while if I have to run through the entire thing from scratch. Having the what changed portion would be very helpful.